sync2jira/upstream_issue.py

```diff
@@ -358,6 +361,25 @@ def github_issues(upstream, config):
         if issue.get('milestone', None):
             issue['milestone'] = issue['milestone']['title']
 
+        if not issue.get('storypoints', None):
+            issue['storypoints'] = ''
+            graphqlurl = 'https://api.github.com/graphql'
+            orgname = "open-connectors"
+            reponame = "open-connectors"
```

Comment:
`graphqlurl` should probably be defined statically (e.g., as a module-level definition or in some sort of external configuration).

`orgname` and `reponame` should be defined externally.

Reply:
Done. 46248b8bbea26b8508001cedd23126f6719599b9